src/Processors/Formats/RowInputFormatWithNamesAndTypes.h

```diff
@@ -33,6 +33,9 @@ class RowInputFormatWithNamesAndTypes : public RowInputFormatWithDiagnosticInfo
     void resetParser() override;
 
 protected:
+    void readPrefix() override;
+    bool readRow(MutableColumns & columns, RowReadExtension & ext) override;
```

Comment:
Now we don't need to put these methods in `protected` part.

Reply:
done